octobot_trading/exchanges/exchange_builder.py

```diff
@@ -63,7 +69,10 @@ async def _build_exchange_manager(self):
 
             # create trading modes
             if self._is_using_trading_modes:
-                self.exchange_manager.trading_modes = await self._build_modes()
+                if self.exchange_manager.is_trading:
+                    self.exchange_manager.trading_modes = await self._build_trading_modes(trading_mode_class)
+                else:
+                    self.logger.info(f"{self.exchange_name} exchange is online and won't be trading")
```

Comment:
What do you think about "exchange is online **but** won't be trading"

Reply:
I was also thinking about this option and at the end I think **and** is better because it's a normal behaviour, it's not like if there is an issue somewhere. Usually adding **but** means something is negative, here it's not the case.